tests/ViewHelpers/ApplicantFormViewHelperTest.php

```diff
@@ -18,13 +18,15 @@ public function testSuccessApplicantForm()
             'stringThree' => ['id' => 1, 'option' => 'dummy-option', 'stageId' => 1]
         ];
 
+        $currentStage = ['dummy-selected'];
```

Comment:
need to change this to match one of the stages that are defined here, currently no stage has this name

Reply:
its done